### DISCUSSION TITLE
Eager Evaluation

### DISCUSSION THREAD

**Discussion by jasonLaster** [RecordReplay/devtools#4077] [Ideas]
Eager evaluation is a feature in chrome and firefox devtools where the JS Engine eagerly evaluates an expression as the user is typing up until the point where there is a mutation.

This makes the console feel much more interactive. It could also speed up logpoints and conditionals as well.

https://developer.chrome.com/blog/new-in-devtools-68/#eagerevaluation

tracking backend work [here](https://github.com/RecordReplay/backend/issues/3294)

**Comment by jonbell-lot23** [comment]
March 15, 2022:

We shipped this, yes?
